Sound.py

```diff
@@ -0,0 +1,497 @@
+#import os, sys
+import simpleaudio as sa
+# import numpy as np
+import wave
+import JESConfig
+from SoundSample import SoundSample
+#import FileChooser
+
+class Sound:
+    MAX_NEG = -32768
+    MAX_POS = 32767
+    SAMPLE_RATE = 22050
+    NUM_BITS_PER_SAMPLE = 16
+    _SoundIndexOffset = 0
+
+    def __init__(self, sound=None, sampleRate=22050):
+        """Construct new sound object
+        
+        If first passed parameter is the name of a WAV file, then read
+        the file.
+
+        If first passed parameter is an int representing a number of
+        frames, then construct Sound object of the specified length.
+        This sound will simply consist of an empty byte array and an
+        AudioFileFormat with the following values:
+            AudioFileFormat.Type.WAVE</code>
+            22.05K sampling rate
+            16 bit sample
+            1 channel
+            signed PCM encoding
+            small-endian byte order
+        Note that no new sound file is created, we only represent the
+        sound with a buffer and the AudioFileFormat.  If a file is
+        desired, then the method writeToFile(filename) must be called
+        on this newly created sound.
+    
+        Parameters
+        ----------
+        sound : str
+            the filename of containing a sound in WAV format
+        sound : int
+            the number of samples in the sound
+        sound : Sound
+            a preexisting Sound object
+        sampleRate : int
+            the frame rate for the sound
+        """
+        if (sound == None):
```

Comment:
I don't think this will work.  We need to call Sound's __init__() with SAMPLE_RATE*3 as an argument.  Also, the following "if" should be an "elif"

Reply:
I fixed this, pushed it, and am about to merge this in :)